Xamarin.Forms.Xaml.UnitTests/Issues/Bz53561.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
you wouldn't get any release at all if our unit tests projects were failing to build, or tests were failing to pass